scripts/missions/amk/07_Shock_Arrant_Abuse_of_Authority.lua

```diff
@@ -6,8 +6,6 @@
 -- Note: KI aquisition is handled in chocobo_digging.lua
```

Comment:
I am going to be removing the 32001 event processing from the existing battlefield (below).  Please make sure to set an onEventFinish for 32001 to handle things.

```lua
    if csid == 32001 then
        if player:getCurrentMission(xi.mission.log_id.ACP) == xi.mission.id.acp.THOSE_WHO_LURK_IN_SHADOWS_III then
            player:completeMission(xi.mission.log_id.ACP, xi.mission.id.acp.THOSE_WHO_LURK_IN_SHADOWS_III)
            player:addMission(xi.mission.log_id.ACP, xi.mission.id.acp.REMEMBER_ME_IN_YOUR_DREAMS)
        end

        if
            not player:hasKeyItem(xi.ki.IVORY_KEY) and
            player:getCurrentMission(xi.mission.log_id.ACP) >= xi.mission.id.acp.THOSE_WHO_LURK_IN_SHADOWS_III
        then
            player:setCharVar('LastIvoryKey', getMidnight())
            player:addKeyItem(xi.ki.IVORY_KEY)
            player:messageSpecial(ID.text.KEYITEM_OBTAINED, xi.ki.IVORY_KEY)
        end
    end
```